src/OrefAlertsRetrieve.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import warnings
+warnings.filterwarnings('ignore')
+import requests
+import json
+from datetime import datetime, timedelta
+import time 
+from Errors import RetrieveDataException
+URL = "https://www.oref.org.il//Shared/Ajax/GetAlarmsHistory.aspx"
+
+DAYS_INTERVAL = 2
+CSV_FILE_PATH = "C:/Projects/Python/Projects/BestTimeToShower/data/data.csv"
```

Comment:
use relative file path. not absulute.
in this case:

```suggestion
CSV_FILE_PATH = "../data/data.csv"
```

Reply:
also rename to ALERTS_DATA_FILE
we can tell it's a csv file by the suffix. the name should imply what does is the content of the file